editor/editor_node.cpp

```diff
@@ -5694,6 +5700,7 @@ EditorNode::EditorNode() {
 	p->add_separator();
 	p->add_submenu_item(TTR("Open Recent"), "RecentScenes", FILE_OPEN_RECENT);
 	p->add_separator();
+	p->add_shortcut(ED_SHORTCUT("editor/quick_open", TTR("Quick Open..."), KEY_MASK_CMD + KEY_K), FILE_QUICK_OPEN);
```

Comment:
Seems good to me, but I'd make the shortcut Shift+Alt+O to match the existing shortcuts for Quick Open Scene and Quick Open Script.